libs/design-system/src/molecules/IFramedSidePanel/IFramedSidePanel.tsx

```diff
@@ -106,10 +125,10 @@ export const IFramedSidePanel = ({
       // To avoid this, we're using fixed width and height
       // However, this remains true for breakpoints or default tailwind classes
       className={cn(
-        'plasmo-fixed plasmo-top-0 plasmo-min-h-full plasmo-w-[296px] plasmo-2xl:w-[396px] plasmo-bg-white plasmo-z-10 plasmo-shadow-2xl plasmo-overflow-scroll',
-        isRight ? 'plasmo-right-0 plasmo-rounded-l-md' : 'plasmo-left-0 plasmo-rounded-r-md',
+        'plasmo-fixed plasmo-top-3 plasmo-h-[calc(100%-1.5rem)] plasmo-bg-transparent plasmo-w-[346px] plasmo-2xl:w-[396px] plasmo-z-10 plasmo-shadow-2xl plasmo-overflow-scroll plasmo-rounded-md',
         isOpen ? '' : 'plasmo-hidden'
       )}
+      style={{ left: `${sidePanelLeftPosition}px` }}
```

Comment:
I would add a blocking condition here, something like:
```js
style={{ left: `${Math.min(Math.max(0, sidePanelLeftPosition), window.innerWidth - 356)}px`}}
```
to avoid the issue of the iFrame going out of the window and not being able to catch it back:

![image](https://github.com/Bastien-and-Gauvain/monorepo/assets/45038783/15b22ac4-cecc-4a11-842f-7502ed99bdeb)

Reply:
(that being said, gorgeous feature! Very user friendly, I love it and so will users who've already complained about the iframe hiding important stuff on the page)